packages/spacecat-shared-rum-api-client/test/fixtures/bundles-for-form-vitals.json

```diff
@@ -63271,6 +63271,59 @@
       "userAgent": "desktop:windows",
       "weight": 10,
       "events": []
+    },
+    {
+      "id": "fafa",
```

Comment:
how are we exactly verifying that these events are not considered in the final calculations?